eclipse/src/saros/ui/handlers/toolbar/ConsistencyHandler.java

```diff
@@ -368,32 +378,10 @@ public void run() {
               newDirection = FADE_UP;
             }
 
-            setImageDescriptor(new MemoryImageDescriptor(modifyAlphaChannel(OUT_SYNC, newValue)));
+            // TODO: create modified icon
```

Comment:
Nope, was already done. Fix it!

Reply:
Have not tested it but I guess it could work:

```
consistencyToolItem.setIconURI(null) // surpress the default renderer
((ToolItem)consistencyToolItem.getWidget()).setImage(....)
```

I do not know if the setImage call will make a copy of the image, otherwise you have to use 2 images and flip between them.